airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -60,10 +60,6 @@
   dockerImageTag: 1.2.9
   documentationUrl: https://docs.airbyte.com/integrations/destinations/bigquery
   icon: bigquery.svg
-  normalizationConfig:
```

Comment:
This PR https://github.com/airbytehq/airbyte/pull/20640 has fixed this problem. You may want to rebase these changes on master to keep the changes in this PR clean

Reply:
OK. I see it was merged yesterday.
Updating